src/containers/pages/InterventionPlan/NewPlan/test/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,238 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`containers/pages/NewPlan renders correctly: page title 1`] = `
+<h3
+  className="mb-3 page-title"
+>
+  Create New Plan
+</h3>
+`;
+
+exports[`containers/pages/NewPlan renders correctly: plan form props 1`] = `
+Object {
+  "allFormActivities": Array [
```

Comment:
Can we make this comparison against a fixture as opposed to have it in the snapshot?

Something like:

```ts
expect(wrapper.find('PlanForm').props()).toEqual(fixtures.theExpoectedProps);
```